packages/tools/examples/planarFreehandContourSegmentationTool/index.ts

```diff
@@ -0,0 +1,580 @@
+  toolGroup.setToolActive(StackScrollMouseWheelTool.toolName);
+
+  // set up toggle interpolation tool button.
+  addToggleInterpolationButton(toolGroup);
+
+  // set up toggle calculate stats tool button.
+  addToggleCalculateStatsButton(toolGroup);
+
+  // Get Cornerstone imageIds and fetch metadata into RAM
+  const stackImageIds = await createImageIdsAndCacheMetaData({
+    StudyInstanceUID:
+      '1.3.6.1.4.1.14519.5.2.1.7009.2403.334240657131972136850343327463',
+    SeriesInstanceUID:
+      '1.3.6.1.4.1.14519.5.2.1.7009.2403.226151125820845824875394858561',
+    wadoRsRoot: 'https://d3t6nz73ql33tx.cloudfront.net/dicomweb',
+  });
+
+  // Define a stack containing a single image
+  const smallStackImageIds = [stackImageIds[0], stackImageIds[1]];
+
+  const volumeImageIds = await createImageIdsAndCacheMetaData({
+    StudyInstanceUID:
+      '1.3.6.1.4.1.14519.5.2.1.7009.2403.334240657131972136850343327463',
+    SeriesInstanceUID:
+      '1.3.6.1.4.1.14519.5.2.1.7009.2403.226151125820845824875394858561',
+    wadoRsRoot: 'https://d3t6nz73ql33tx.cloudfront.net/dicomweb',
+  });
+
+  // Instantiate a rendering engine
+  const renderingEngine = new RenderingEngine(renderingEngineId);
+
+  // Create a stack and a volume viewport
+  const viewportInputArray = [
+    {
+      viewportId: viewportIds[0],
+      type: ViewportType.STACK,
+      element: element1,
+      defaultOptions: {
+        background: <Types.Point3>[0.2, 0, 0.2],
+      },
+    },
+    {
+      viewportId: viewportIds[1],
+      type: ViewportType.ORTHOGRAPHIC,
+      element: element2,
+      defaultOptions: {
+        orientation: Enums.OrientationAxis.SAGITTAL,
+        background: <Types.Point3>[0.2, 0, 0.2],
+      },
+    },
+  ];
+
+  renderingEngine.setViewports(viewportInputArray);
+
+  // Set the tool group on the viewport
+  viewportIds.forEach((viewportId) =>
+    toolGroup.addViewport(viewportId, renderingEngineId)
+  );
+
+  // Define a volume in memory
+  const volume = await volumeLoader.createAndCacheVolume(volumeId, {
+    imageIds: volumeImageIds,
+  });
+
+  // Get the viewports that were just created
+  const stackViewport = <Types.IStackViewport>(
+    renderingEngine.getViewport(viewportIds[0])
+  );
+  const volumeViewport = <Types.IVolumeViewport>(
+    renderingEngine.getViewport(viewportIds[1])
+  );
+
+  // Set the stack on the viewport
+  stackViewport.setStack(smallStackImageIds);
+
+  // Set the volume to load
+  volume.load();
+
+  // Set the volume on the viewport
+  volumeViewport.setVolumes([{ volumeId }]);
+
+  // Render the image
+  renderingEngine.renderViewports(viewportIds);
+
+  // Add a segmentation that will contains the contour annotations
+  segmentation.addSegmentations([
+    {
+      segmentationId,
+      representation: {
+        type: csToolsEnums.SegmentationRepresentations.Contour,
+        data: {
+          geometryIds: [],
+          annotationUIDsMap: new Map(),
+        },
```

Comment:
This should be default handled internally, no? It is very odd that we hare having this new Map here ampty and geometryIds empty